src/platform/Server.c

```diff
@@ -40,19 +54,15 @@ static MPI_HANDLE CallMpiOpen(const char* clientName, const unsigned int maxPayl
 
     if (NULL == (handle = MpiOpen(clientName, maxPayloadSizeBytes)))
     {
-        OsConfigLogError(GetPlatformLog(), "MpiOpen failed to create a session for client '%s'", clientName);
+        LOG_ERROR("MpiOpen failed to create a session for client '%s'", clientName);
     }
```

Comment:
Revert back all changes to this file (and rename it back as it was). There is no reason to refactor anything in here. Refactor the rest to match and complete this file and Main.c.